core/src/test/scala/stryker4s/mutants/findmutants/FileCollectorTest.scala

```diff
@@ -215,18 +215,17 @@ class FileCollectorTest extends Stryker4sSuite with MockitoSugar with LogMatcher
       results should contain theSameElementsAs expectedFileList
     }
 
-    it("Should copy all files when no files config key is found and target repo is not a git repo") {
+    it("Should copy not copy files our of the target folders when no files config key is found and target repo is not a git repo") {
```

Comment:
Should copy not copy? 🤔

Reply:
if only i had a deep learning grid that knows how i write my danglish and fixes it for me automaticly